src/utilities/conditionals.ts

```diff
@@ -6,3 +8,20 @@
 export function reform2005Applies(year: number): boolean {
     return year >= 2005;
 }
+
+/**
+* Helper function to check if settings have changed.
+* @param settingsPayload The current chosen settings
```

Comment:
Codacy found an issue: [asterisks in jsdoc must be aligned](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=6071614)